SU-SWS/stanford_basic
# Superfluous Layout Templates

# READY FOR REVIEW

# Summary
- We are not using these templates and don't plan to. 

# Review By (Date)
- Whenever

# Urgency
- Super low. They are doing nothing at the moment.

# Steps to Test

1. Check out this branch.
2. Clear cache.
3. Navigate to a place where you can select layouts in the UI.
4. Rejoice that they are still there because they are in jumpstart_ui
5. For bonus points, remove the complex layouts from jumpstart_ui and leave the simple ones.

# Affected Projects or Products
- Not much.

# Associated Issues and/or People
- None

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
